example/test/readme_test.dart

```diff
@@ -15,14 +15,14 @@ void _expect(String fileName) {
   test(fileName, () {
     final file = File(fileName);
     expect(file.readAsStringSync(), contains(_pubspecContent));
-  });
+  }, skip: true);
```

Comment:
You have to copy the contents of the doc/...whatever file into the readme. This makes sure all of the options are linked!

Reply:
Did you rerun the build?

Reply:
Oh wait, sorry. Hrm. Let me dig in here. Sorry.

Reply:
I've now run the rebuild and commited the generated files.
I had to go through and rationalise the job sdk versions. I've changed them all to stable.
We had a mix of 2.6 and dev which I don't think is necessary now that 2.7 is out.

Everything now appears to be passing except for the readme_test but I can't see how that ever worked as noted previously,

Reply:
worked out the readme problem. the example.g.dart contents needed to be copied into the readme. The test is not an 'equals' but a 'contains' so it does actually work as published.